rasa/train.py

```diff
@@ -396,14 +399,15 @@ async def _train_core_with_validated_data(
         domain, config = await asyncio.gather(
             file_importer.get_domain(), file_importer.get_config()
         )
-        await rasa.core.train(
-            domain_file=domain,
-            training_resource=file_importer,
-            output_path=os.path.join(_train_path, DEFAULT_CORE_SUBDIRECTORY_NAME),
-            policy_config=config,
-            additional_arguments=additional_arguments,
-            interpreter=interpreter,
-        )
+        async with telemetry.track_model_training(file_importer, model_type="core"):
```

Comment:
when training a model, this will track 3 times: 
1. the entire training
2. the core training
3. the NLU training

Reply:
yes, this is intentional

Reply:
What's the intention behind it?

Reply:
we get runtimes for the separate parts and for the whole training (and e.g. model sizes if we want to add that)

Reply:
Makes sense 👍